dashboard/modules/stats_collector/stats_collector_head.py

```diff
@@ -10,32 +10,44 @@
 import ray.new_dashboard.modules.stats_collector.stats_collector_consts \
     as stats_collector_consts
 import ray.new_dashboard.utils as dashboard_utils
+from ray.new_dashboard.actor_utils import actor_classname_from_task_spec
 from ray.new_dashboard.utils import async_loop_forever
 from ray.new_dashboard.memory_utils import GroupByType, SortingType
 from ray.core.generated import node_manager_pb2
 from ray.core.generated import node_manager_pb2_grpc
 from ray.core.generated import gcs_service_pb2
 from ray.core.generated import gcs_service_pb2_grpc
 from ray.new_dashboard.datacenter import DataSource, DataOrganizer
-from ray.utils import binary_to_hex
 
 logger = logging.getLogger(__name__)
 routes = dashboard_utils.ClassMethodRouteTable
 
 
 def node_stats_to_dict(message):
-    return dashboard_utils.message_to_dict(
-        message, {
-            "actorId", "jobId", "taskId", "parentTaskId", "sourceActorId",
-            "callerId", "rayletId", "workerId"
-        })
+    decode_keys = {
+        "actorId", "jobId", "taskId", "parentTaskId", "sourceActorId",
+        "callerId", "rayletId", "workerId", "placementGroupId"
+    }
+    core_workers_stats = message.core_workers_stats
+    message.ClearField("core_workers_stats")
+    try:
+        result = dashboard_utils.message_to_dict(message, decode_keys)
```

Comment:
This change solved this contradiction,

1. We want the `coreWorkerStats` including full fields, so that we don't needs to fill default fields manually.
2. We only needs non-default fields of `view_data`.